lesson_23/script.js

```diff
@@ -0,0 +1,53 @@
+let num = [117, 15, 100000, 1, 55, "0", 7, false, 578, "5896777", 1001, "якесь число", 2510, true, 3512];
+
+// Сума елементів масиву з перевіркою на число =======================
+
+console.log("_______Сума елементів масиву________");
+
+let sum = 0;
+
+for (let i = 0; i < num.length; i++) {
+    if (typeof num[i] === "number") {
```

Comment:
should also check `!isNaN(num[i])` as NaN also return type `number`

Reply:
Yes, i wanted to make it, but it worked without this and i decided don't make. I've understand. Thanks